.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libafl_concolic/libafl_symcc"]
```

Comment:
Should we eventually pull this in via Cargo? Or will that not work because it needs to be built using cmake?

Reply:
I am honestly not sure how exactly this should _ideally_ work. I'll need to gather some info an write up a more detailed explanation for a proper discussion soon. the short version is this: when you compile symcc, the executables compiled with symcc are (dynamically) linked against the symcc runtime (which we implement partially in rust).  this 'links' a symcc build to its runtime, which makes things a bit complicated.

In the end, it would be awesome if we could have something like afl_cc, just with symcc. so essentially, a build_script type crate that bundles symcc so that the user can build an instrumented binary simply using a build script like in the example fuzzers. however, since the symcc build and runtime are linked, this compiler crate would also depend on the implementation of the runtime, which, _ideally_ would be implemented by the user in a separate crate and pulled into the compiler wrapper crate. at this level of decency mess my head explodes a little, which is why I haven't come up with a proper plan yet...

as for right now: I'd like to keep it as a submodule as it makes development easy.